README.md

```diff
@@ -551,6 +551,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [PyCharm](https://www.jetbrains.com/pycharm/)
 - [PyPI](https://pypi.python.org/pypi)
 - [PyRadio](http://www.coderholic.com/pyradio/)
+- macOS Preview
```

Comment:
Can you keep the list AZ ordered?

Reply:
https://github.com/lra/mackup/pull/1881/commits/114cf0cc48aad51afc6574a2bf0b1941d212d3be